internal/api/enclosure.go

```diff
@@ -0,0 +1,85 @@
+// SPDX-FileCopyrightText: Copyright The Miniflux Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package api // import "miniflux.app/v2/internal/api"
+
+import (
+	json_parser "encoding/json"
+	"net/http"
+
+	"miniflux.app/v2/internal/http/request"
+	"miniflux.app/v2/internal/http/response/json"
+	"miniflux.app/v2/internal/model"
+	"miniflux.app/v2/internal/validator"
+)
+
+func (h *handler) getEnclosureById(w http.ResponseWriter, r *http.Request) {
+	enclosureID := request.RouteInt64Param(r, "enclosureID")
+
+	enclosure, err := h.store.GetEnclosure(enclosureID)
```

Comment:
GetEnclosure is fixed now, but I would like to add integration tests if I'm to extend the client as well because I'm not using that codebase myself. Any chance I could get an enclosure somewhere in the Miniflux RSS feed that's being used for integration testing @fguillot?

Reply:
Sorry forgot to push the mentioned fix. It's pushed now though, and the client has been updated with Enclosure and UpdateEnclosure. Both methods have integration tests which skips if no enclosures are found in the test feed.